lte/gateway/python/magma/subscriberdb/protocols/m5g_auth_servicer.py

```diff
@@ -176,14 +176,29 @@ def M5GDecryptImsiSUCIRegistration(self, request, context):
                 profile,
             )
 
-            msin_recv = home_network_info.unprotect(
-                request.ue_pubkey, request.ue_ciphertext,
-                request.ue_encrypted_mac,
-            )
+            if len(request.ue_pubkey) != 32:
+                set_grpc_err(
+                    context,
+                    StatusCode.NOT_FOUND,
```

Comment:
Seems more like an INVALID_ARGUMENT to me?

Reply:
Done